notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<p>It would be nice to have an introductory paragraph here that explains the file formats we support and where to find more information about those formats.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I only talked about feather, I think that is all that work now right?

Reply:
I think so, yes.